sunpy/timeseries/sources/noaa.py

```diff
@@ -59,60 +60,89 @@ class NOAAIndicesTimeSeries(GenericTimeSeries):
     # Class attribute used to specify the source class of the TimeSeries.
     _source = 'noaaindices'
 
-    @peek_show
-    def peek(self, type='sunspot SWO', **kwargs):
+    def plot(self, title='Solar Cycle Progression', plot_type='sunspot SWO', dataframe=None, **kwargs):
```

Comment:
I know I wrote this but I do wonder if this should just output all of the plot types in big subplot.

Reply:
Should I make that change?

Reply:
It might be, I will let others have a review and decide.